packages/run-it/src/components/ConfigForm/ConfigForm.tsx

```diff
@@ -130,43 +128,51 @@ export const ConfigForm: FC<ConfigFormProps> = ({
       baseUrl: base_url,
       webUrl: looker_url,
       [fetchIntent]:
-        base_url !== '' && looker_url !== '' ? 'positive' : 'critical',
+        base_url !== '' && looker_url !== '' ? positive : 'critical',
     }))
   }, [configurator])
 
   const [validationMessages, setValidationMessages] =
     useState<ValidationMessages>({})
 
+  const updateFetch = (intent: MessageBarIntent, message: string) => {
```

Comment:
is updateMessage a better name?

Reply:
Yes it absolutely is. I'll make that change tomorrow too.